include/common/complex16x2simd.hpp

```diff
@@ -12,6 +12,8 @@
 
 #pragma once
 
+#define ZERO_256D _mm256_set_pd(0, 0, 0, 0)
```

Comment:
The changes in this file really seem like they should be part of a separate PR.

Reply:
The incompatibility with Microsoft was discovered in the course of building the DLL. Please, they're Windows-specific compatibility changes.